CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 # Release History
-## 0.9.0 (Unreleased)
+## 0.9.0 (2022-08-31)
 
 ### New Features:
 - Added support for displaying source code as comments in the generated scripts when registering UDFs.
 This feature is turned on by default. To turn if off, pass the new keyword argument `source_code_display` as `False` when calling `register()` or `@udf()`.
+- Added support for `column_names` in `df.write.save_as_table()`. You can now specify the names of columns when appending new data to an existing table.
```

Comment:
This is not accurate. A new param `column_order` is added. When it's "name" and `save_as_table` is in append mode, the dataframe data is appended to an existing table by matching the dataframe's column names and the table's column names. When it's "index", it works as before.

Reply:
my fault, I read into the analyzer part /facepalm/.

I have updated to `Added keyword argument `column_order` to `df.write.save_as_table()` to specify the matching rules when inserting data into table in append mode.`

how does it look to you now?